src/views/open-api-docs-view/components/operation-sections/operation-sections.module.css

```diff
@@ -4,24 +4,41 @@
  */
 
 .root {
-	display: flex;
-	flex-direction: row-reverse;
-	flex-wrap: wrap;
+	/* stylelint-disable-next-line property-no-unknown */
+	container-type: inline-size;
+}
+
+.gridLayout {
+	display: grid;
```

Comment:
TLDR; is i've switched from a `flex` layout to a `grid` layout with a container query! ✨ I had to add some `stylelint` exceptions cause container CSS didn't seem to be recognized yet by linting.

Reply:
Whoa container query?! ✨

Reply:
It's a new direction for sure! I am so used to thinking container queries will never be a thing that I have not kept up at all with browser support, but as [Dylan mentioned in Slack](https://hashicorp.slack.com/archives/C031LA4KS4S/p1689708868902629):

> It's been five months since the last major browser engine added support, so I don't think we need to worry about polyfills or anything. I believe our official support matrix is the last two versions of evergreen browsers.